src/Raven.Server/ServerWide/Context/DocumentsOperationContext.cs

```diff
@@ -12,7 +12,18 @@ public class DocumentsOperationContext : TransactionOperationContext<DocumentsTr
 
         internal string LastDatabaseChangeVector;
         internal Dictionary<string, long> LastReplicationEtagFrom;
-        internal bool SkipChangeVectorValidation;
+        private bool _skipChangeVectorValidation;
+
+        internal bool SkipChangeVectorValidation
```

Comment:
I'm not seeing usage of this?

Reply:
This is for debug purpose to assert that we don't move the global change vector back in time.